board/piksiv3/rootfs-overlay/etc/init.d/S85piksi_firmware

```diff
@@ -79,4 +86,3 @@ stop()
 }
 
 source /etc/init.d/template_command.inc.sh
```

Comment:
Can you make sure that this and any other whitespace / style only changes are in a separate commit when you merge?

Reply:
Done

Reply:
👍